fbpic/boundaries/moving_window.py

```diff
@@ -33,6 +33,14 @@ def __init__( self, interp, comm, ptcl, v, p_nz, time,
             Contains information about the MPI decomposition
             and about the longitudinal boundaries
 
+        exchange_period: int
```

Comment:
Shouldn't we remove ` after which the moving window is updated`, since the moving window is now updated at every single timestep ?